sourmash_lib/__main__.py

```diff
@@ -22,7 +22,7 @@
 class SourmashCommands(object):
     commands = ['search', 'compute', 'compare', 'plot', 'import_csv',
                 'dump', 'sbt_index', 'sbt_search', 'categorize', 'sbt_gather',
-                'watch']
+                'watch', 'convert']
```

Comment:
An other naming options for the action could be:
- `upgrade_signature`
- `upgrade_to_v2`